etc/install_package_dependencies.sh

```diff
@@ -1,30 +1,28 @@
-# This script can be used to install all the required package dependencies on a clean system
-
+#This script can be used to install all the required package dependencies on a clean system
+ 
 PROJECT_PATH=$(pwd)
 
 # Install Python-PIP
 sudo apt-get update
-sudo apt-get install -y python-pip
+sudo apt-get install -y python3-pip
+
+# Install virtualenv, create a virtual environment and activate it
+sudo -H pip3 install virtualenv
+virtualenv env
+source env/bin/activate
```

Comment:
can the virtualenv be installed in .env instead of env?

Reply:
The idea makes sense. Also the `.env` directory needs to be added to `.gitignore`.

Reply:
.env directory already present in .gitignore